packages/server/src/restaurants/profile/domain/UpdatePartnerProfileHandler.ts

```diff
@@ -26,10 +26,7 @@ class UpdatePartnerProfileHandler implements Handler<UpdatePartnerProfileRequest
       {
         name: req.name,
         description: req.description,
-
-        // TODO: add logo
-        // logo: {},
-
+        logo: req.logo,
```

Comment:
IMO to powinno być całkiem usunięte + po prostu zmienić
```diff
- class UpdatePartnerProfileDto extends OmitType(PartnerProfileDto, ['id'] as const) {}
+ class UpdatePartnerProfileDto extends OmitType(PartnerProfileDto, ['id', 'logo'] as const) {}
```

Teraz partner może sobie np. ustawić logo `https://myfakemaliciouswebsite.com/api/img/restaurant/6200218668fc82e7bdf15088` i nie wiem po co nam taka funkcjonalność (+ wtedy nie możemy wykorzystać kilku uproszczeń, np. że logo restauracji jest zawsze pod adresem `https://coderscamp2021-hk-fullstack.herokuapp.com/api/img/restaurant/<restaurantId>`, więc na froncie możemy ustawić `<img src="...">` nawet bez requesta do API).